### ISSUE TITLE
If aioredis.create_redis_pool encoding parameter is set, aioredlock raise AttributeError ?

### ISSUE THREAD

**Issue by trankos** [joanvila/aioredlock#86]
### Error trace

>  aioredlock/redis.py", line 121, in <genexpr> ) = (r.decode() for r in await asyncio.gather(*tasks))
>      AttributeError: 'str' object has no attribute 'decode'

### Probable cause
```
redis_lock = await aioredis.create_redis_pool(address=REDIS_LOCK_ADDRESS, encoding='utf-8')
lock_manager = Aioredlock([redis_lock])
async with await lock_manager.lock() as lock:
       assert lock.valid is True
       ....
```
-  If enconding is set (utf-8), task redis.script_load() returns string type (decode method does not exist)
-  If encoding is not set, redis.script_load() returns bytes type ( decode method exists)

**Comment by gtmanfred** [comment]
Thanks for reporting this. I will take a look.

**Comment by gtmanfred** [comment]
Sorry it took so long, fixing this today.

**Comment by gtmanfred** [comment]
This is fixed in v0.7.1, thank you for reporting the bug.
